website/scripts/ros/ros_host_vehicle_cmd.js

```diff
@@ -65,3 +65,60 @@ function subscribeToVehicleCMD()
         }
     });
 }
+
+/**
+ * /hardware_interface/speed_pedals
+ */
+function subscribeToSpeedPedals()
+{
+    var listener = new ROSLIB.Topic({
+        ros: g_ros,
+        name: "/hardware_interface/speed_pedals",
+        messageType: "automotive_platform_msgs/SpeedPedals"
+    });
+    listener.subscribe(function(message)
+    {
+        //Check ROSBridge connection before subscribe a topic
+        IsROSBridgeConnected();
+        if(message!=null && message.brake !=null)
+        {
+              //Brake(0-1 percent)
+              if(message.brake != null && g_brakeCircle != null)
+              {
+                let max, brakeLimit = '';
+                //set brake  limit to the host vehicle deceleration(brake) limit if exist in session, otherwise default to 6
+                if(message.brake>0)
+                    brakeLimit = session_hostVehicle.brakeLimit;
+                  
+                max =  brakeLimit != null && brakeLimit.length > 0? brakeLimit: 6;
+                let value = message.brake * max; //brake value
+                updateBrake(g_brakeCircle,max,value);
+              }
+        }
+    });
+}
+/**
+ * /hardware_interface/steering_wheel 
+ */
+function subscribeToSteeringWheel()
+{
+    var listener = new ROSLIB.Topic({
+        ros: g_ros,
+        name: "/hardware_interface/steering_wheel",
+        messageType: "automotive_platform_msgs/SteerWheel"
+    });
+    listener.subscribe(function(message)
+    {
+        //Check ROSBridge connection before subscribe a topic
+        IsROSBridgeConnected();
+        //Steering wheel
+        if(message.angle != null)
+        {
+            let rad = message.angle * 180/(Math.PI);
```

Comment:
This variable should be named 'deg' since its units are in degrees not radians after applying the conversion